acceptance/bundle/resources/grants/schemas/empty_array/out.deploy1.requests.direct.json

```diff
@@ -0,0 +1,14 @@
+{
+  "method": "PATCH",
+  "path": "/api/2.1/unity-catalog/permissions/schema/main.schema_grants_[UNIQUE_NAME]",
+  "body": {
+    "changes": [
+      {
+        "principal": "deco-test-user@databricks.com",
+        "remove": [
```

Comment:
Where does the remove `ALL_PRIVILEGES` come from?

Reply:
See grants.go